src/client/render/plain.cpp

```diff
@@ -115,13 +116,18 @@ RenderStep* addUpscaling(RenderPipeline *pipeline, RenderStep *previousStep, v2f
 	if (g_settings->getBool("enable_shaders") && g_settings->getBool("enable_post_processing"))
 		return previousStep;
 
+	auto driver = client->getSceneManager()->getVideoDriver();
```

Comment:
fyi `RenderingEngine::get_video_driver()` also works

Reply:
yeah, I did it this way for consistency with the rest of the rendering pipeline code